templates/solutions/ai-infrastructure/index.html

```diff
@@ -0,0 +1,459 @@
+  </section>
+
+  <section class="p-section">
+    <div class="p-section--shallow">
+      <div class="row--50-50">
+        <hr class="p-rule" />
+        <div class="col">
+          <h2>Deploy on any platform, scale with a hybrid cloud strategy</h2>
+        </div>
+        <div class="col">
+          <p>
+            Choose the ideal AI infrastructure for your use cases &mdash; for instance, start quickly with no risk and low investment on public clouds, then move workloads to your own data centre as you scale.
+          </p>
+          <p class="p-section--shallow">
+            With Canonical’s solutions, you can run your workloads anywhere, including hybrid and multi-cloud environments.
+          </p>
+          <hr class="is-muted" />
+          <a href="https://ubuntu.com/engage/hybrid-cloud-strategy-playbook">Download our hybrid cloud strategy playbook&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="p-section--shallow">
+      <div class="row--50-50">
+        <hr class="p-rule" />
+        <div class="col">
+          <h2>Deploy machine learning models to the edge</h2>
+        </div>
+        <div class="col">
+          <p>
+            Unlock real-time data processing in distributed environments by deploying machine learning models to your edge devices.
+          </p>
+          <p class="p-section--shallow">Canonical infrastructure spans the entire AI journey, from the desktop to the edge.</p>
+          <hr class="is-muted" />
+          <a href="https://ubuntu.com/engage/guide-to-deploying-ai-at-the-edge">Access our guide to open source edge AI&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="p-section--shallow">
+      <div class="row--50-50">
+        <hr class="p-rule" />
+        <div class="col">
+          <h2>
+            Stay in control
+            <br />
+            of AI infrastructure costs
+          </h2>
+        </div>
+        <div class="col">
+          <p>
+            Data volumes involved in AI projects can scale rapidly, making public cloud prohibitively costly. If operated efficiently, a private cloud is more cost-effective for running workloads long-term and at scale.
+          </p>
+          <p class="p-section--shallow">
+            Build your private cloud with our AI infrastructure solutions and enjoy predictable pricing per node with no licence fees.
+          </p>
+          <hr class="is-muted" />
+          <a href="https://ubuntu.com/engage/cloud-pricing-report">Check out our cloud pricing report&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <h2 class="p-text--small-caps">What customers say</h2>
+    </div>
+    <div class="row">
+      <hr class="p-rule" />
+      <div class="col-3 col-medium-2 p-image-wrapper p-section">
+        {{
```

Comment:
Could you apply the same formatting fix here as well?

Reply:
Thanks for the catch!